src/main/java/org/jabref/gui/exporter/SaveDatabaseAction.java

```diff
@@ -160,9 +159,14 @@ private boolean doSave() {
     }
 
     public boolean save() {
+        return save(false);
+    }
+
+    public boolean save(boolean silent) {
```

Comment:
Please, no `boolean` parameters. Please use an Enum. Even `SILENT, LOUD` are as values OK. Boolean should be avoided at (nearly) all costs. See also https://java.by-comparison.com/.

Reply:
Please do not use `boolean` as return value. I think, it is never used, is it

Oh, it is also old code. Maybe, you can at least add a JavaDoc comment?

Reply:
Thanks for the suggestion. I agree that enums are more readable and I have changed that.
I haven't added a JavaDoc, as I don't really know how the method behaves. If you want I can have a deeper look though.